HHS/Head-Start-TTADP
# [Prod] Fix bugs with approval status calculation, resource deduplication

## Description of change
- Fix a bug which was causing AR status to be incorrectly calculated if the approver was added before submission
- Fix a bug which caused only one resource to appear when editing objective resources on the AR

## How to test
- Create a report; save an approver. Fill out the rest of the report and submit. The report should show as "submitted"
- Add multiple resources to an AR objective. They should all still be there after you save

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2494


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
